plugins/gcc10/binutils-1-fixes.patch

```diff
@@ -1,177 +0,0 @@
-This file is part of MXE. See LICENSE.md for licensing information.
-
-Contains ad hoc patches for cross building.
-
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: Stephen Kitt <skitt@debian.org>
-Date: Sat, 15 Jul 2017 00:09:40 +1000
-Subject: [PATCH 1/2] Allow the PE timestamp to be specified with
- SOURCE_DATE_EPOCH
-
-Taken from:
-https://sources.debian.net/patches/binutils-mingw-w64/7.4/specify-timestamp.patch/
```

Comment:
Was this `SOURCE_DATE_EPOCH` patch merged in the binutils source? I thought it was a Debian-specific patch that's part of their reproducible builds project.

Reply:
@v1993 removed this. Can you answer this question?

Reply:
It's doesn't seems to be fixed upstream. I will restore the patch file.

Reply:
@tonytheodore It's now located here: src/binutils-1-fixes.patch

Reply:
I remember it again. I initially made gcc10 the default in this build. But now we made gcc11 the default compiler in MXE Upstream (see PR: https://github.com/mxe/mxe/pull/2755). The active patch file should be in `src/` folder. 

But I think I forgot the restore the gcc10 patch file again, due to all the rebasing between other PRs, master and this branch. Let me restore this file.

Reply:
Small issue at the moment with the binutils patch on binutils v2.37.. Looking into it.